modules/helper/GeneralHelper.py

```diff
@@ -0,0 +1,22 @@
+from interfaces.host import Host
+from interfaces.vcenter import Vcenter
+from modules.configuration.GeneralHelperConfig import GeneralHelperConfig
+from modules.api.Atlas import Atlas
+
+from typing import List
+
+
+class GeneralHelper:
+    """
+    The general helper provides functionality of shared APIs.
+    """
+
+    def __init__(self):
+        config = GeneralHelperConfig()
+        self.atlas = Atlas(config.atlas_file)
+
+    def get_vcenters(self) -> List[Vcenter]:
+        return self.atlas.get_vcenters()
+
+    def get_esxi_hosts(self) -> List[Host]:
+        return self.atlas.get_esxi_hosts()
```

Comment:
Would remove, this is basically just obfuscating your Atlas class without adding any benefit. It makes it harder to read. When you call GeneralHelper you actually want to set `self.atlas`. Why not calling it straight? Only thing which is masked in the end, is the read of the atlas env var.  Put that to the `BaseCollector` `__init__` too and pass it as an argument to `VCenterHelper`.

Remove this class entirely, it has the same function names as the atlas class itself, no functionality is added.